test/unittests/execute_floating_point_test.cpp

```diff
@@ -11,8 +11,11 @@
 #include <test/utils/floating_point_utils.hpp>
 #include <test/utils/hex.hpp>
 #include <array>
+#include <cfenv>
 #include <cmath>
 
+#pragma STDC FENV_ACCESS ON
```

Comment:
I know it is possible to look what this means -- I just did that --, but perhaps nicer to leave a small warning here so that it is not removed in the future:
> Note: this is important because our tests need complete control over floating point settings.

Reply:
I would comment what pragma does (enables access and modification of the floating-point environment (rounding directions in particular) ).

Reply:
Yes, I will add it.

Reply:
Added.